client/app/app.routes.js

```diff
@@ -89,11 +89,10 @@
         },
         resolve: {
           checkUser: ['$http', checkUser],
-          mealsData: ['$http', '$stateParams', function($http, $stateParams) {
```

Comment:
Is there a reason you changed this back? Ideally, you should use the array, just in case we ever planned on minifying.